metal/i2c.h

```diff
@@ -0,0 +1,112 @@
+/* Copyright 2018 SiFive, Inc */
+/* SPDX-License-Identifier: Apache-2.0 */
+
+#ifndef METAL__I2C_H
+#define METAL__I2C_H
+
+/*! @brief Enums to enable/disable stop condition. */
+typedef enum {
+    METAL_I2C_STOP_DS = 0,
+    METAL_I2C_STOP_EN = 1
+} metal_i2c_stop_bit_t;
+
+/*! @brief Enums to set up I2C device modes. */
+typedef enum { METAL_I2C_SLAVE = 0, METAL_I2C_MASTER = 1 } metal_i2c_mode_t;
```

Comment:
consistent format like above

Reply:
formatter gives this output for enum typedefs, if total length is less than <80 chars. re-formatting breaks format checks.

Reply:
The right thing is by-definition whatever `clang-format` does. If we want it to look different we can modify the format configuration.